test/c/test_main.c

```diff
@@ -155,4 +155,4 @@ int main()
     test_num_qubits();
 
     return 0;
```

Comment:
Do you know how we can see if a single test fails? I.e. if we have a `test_sparse_observable.c` file which has 100 tests and there's a single one failing, we'd still like the rest to run. Do we have to build something on our own or is there a functionality for this?

Reply:
Yes that's what I am looking into now. Should add that as a todo in the description